### ISSUE TITLE
Integration to right-click menu

### ISSUE THREAD

**Issue by XYZliang** [zaru/menuffy#4]
In some cases, using shortcut key activation seems a little tedious. It may be more convenient if it can be integrated in the right-click menu.

**Comment by noestreich** [comment]
Or maybe via middle-mouse-click? Anyway a mouse activation method would be appreciated.

**Comment by zaru** [comment]
@XYZliang Does that mean adding it to the normal right-click context menu?

**Comment by XYZliang** [comment]
> @XYZliang 是否意味着将其添加到正常的右键上下文菜单中？

Right. It's cold in winter. When I don't need two hands to control my mac, such as browsing the web page, I leave only one hand with the mouse and the other in my pocket. It may be easier to add to the right-click extension menu of the mac system. It can also be activated by other buttons on the mouse, as Nosetreich said. Of course it's up to you! Thank you for developing menuffy and happy New Year!
